parsers/US_NY.py

```diff
@@ -8,13 +8,21 @@
 import pandas as pd
 
 mapping = {
-           'Dual Fuel': 'unknown',
+           'Dual Fuel': 'gas', # NYISO states "Power produced by facilities capable of using natural gas and/or other fuels"
+                # <5% is oil burn from 2016 State of NYISO Market Report
            'Natural Gas': 'gas',
            'Nuclear': 'nuclear',
-           'Other Fossil Fuels': 'unknown',
-           'Other Renewables': 'unknown',
-           'Wind': 'wind',
-           'Hydro': 'hydro'
+           'Other Fossil Fuels': 'coal', # NYISO states "Power produced by either coal or oil" - assume coal, oil only power plants only as last resort
+           'Other Renewables': 'biomass', # NYISO states "Power produced by either solar energy, refuse, methane or wood"
+                # Based on only 250 MW solar-utility & est 500 MW solar-'behind the meter' (unreportable by NYISO),
+                # and at 0445am NYISO reported 253 MW production --> biomass best assumption
+           'Wind': 'wind', # 2016 Wind capacity is 1754 MW (updated zones.json)
+           'Hydro': 'hydro' # NYISO states "Power produced by water including pumped storage facilities"
+                # hoping for further data source acquistion to distinguish Hydro from PS
```

Comment:
can you remove this comment?

Reply:
-rewrote as information to @systemcatch as the builder of the parser - working on acquiring realtime data from NYPA - NY Power Authority.  They operate 97.7% of all hydro capacity in NY at 3 different locations.